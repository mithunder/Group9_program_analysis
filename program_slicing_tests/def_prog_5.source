module def_prog_5 :
a := 2;
if (true) -> a := 3
[] (true) -> a := 4
[] (true) -> z := 0
fi;
@programslicing="true"
x := a
end
