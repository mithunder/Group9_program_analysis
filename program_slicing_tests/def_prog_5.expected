module def_prog_5 :
a := 2;
if (true) -> a := 3
[] (true) -> a := 4
[] (true) -> skip
fi;
x := a
end
